### PR TITLE
P2022 486 routing to login view

### PR DESCRIPTION
**Pull request by GrzegorzBlack**

### CODE REVIEW

File: src/views/Login/Loginstyle.tsx [hunk 1]
```diff
@@ -0,0 +1,34 @@
+import { Box } from "@mui/material";
```
Comment (KarolCiesluk) [inline src/views/Login/Loginstyle.tsx:1]: Did you mean to name this file `Loginstyle.tsx` or `Login.style.tsx`?

Comment (GrzegorzBlack) [reply src/views/Login/Loginstyle.tsx:1]: It should be LoginView.style.tsx I think.

Comment (GrzegorzBlack) [reply src/views/Login/Loginstyle.tsx]: I fixed it.

File: src/components/PrivateRoute/index.tsx [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+    setLocalStorage(key, defaultValue);
+    return defaultValue;
+  });
+  const setLocalStorageValue = (value: string) => {
+    setValue(value);
+    localStorage.setItem(key, value);
+  };
+
+  return [value, setLocalStorageValue];
+};
```
Comment (jprymak) [inline src/components/PrivateRoute/index.tsx:49]: you could move custom hooks to hooks folder :)

Comment (GrzegorzBlack) [reply src/components/PrivateRoute/index.tsx]: Done.

**Review by jprymak — APPROVED** [review]
Nice

**Review by jprymak — APPROVED** [review]

**Pull request merged**